### PR TITLE
Chen ming

### PR DESCRIPTION
**Pull request by chenming7777**
help

### CODE REVIEW

File: backend/googleTTS.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    return "uploads/audio/fromAI.wav"
+
+
+def main(output_text):
+    audio_paths = tts_process(output_text)
+    return "Audio file created at: " + audio_paths
+
+
+
+main("Hi Chenming 7 7 7 welcome to Hilti AI interview session I'm your AI interviewer EVA, and today I will be interviewing you on business analyst specialize on sustainability. Before we start, please introduce yourself")
```
Comment (SHINE-six) [inline backend/googleTTS.py]: remove this main("")

File: backend/mbti_test.py [hunk 2]
```diff
@@ -69,27 +63,24 @@ def clear_text(data):
...
+    preprocessed_text = pre_process_sentence(text)
     vectorized_text = vectorizer.transform([preprocessed_text])
 
     # Make a prediction
     predicted_class = model_xgb.predict(vectorized_text)
 
     # Translate the numeric prediction to MBTI type
     predicted_mbti = translate_prediction_to_mbti(predicted_class, mbti_mapping)
-    print("Predicted MBTI type:", predicted_mbti)
+    return "Predicted MBTI type:" + predicted_mbti
```
Comment (SHINE-six) [inline backend/mbti_test.py:87]: return a json

File: backend/mbti_test.py [hunk 3]
```diff
@@ -69,27 +63,24 @@ def clear_text(data):
...
+
+# if __name__ == "__main__":
```
Comment (SHINE-six) [inline backend/mbti_test.py:89]: remove

File: backend/requirements.txt [hunk 4]
```diff
@@ -21,7 +21,10 @@ gensim==4.3.2
 # spicy==1.12.0  # this important, 1.13.0 will give error
 scikit-learn==1.2.2 # this important, because the gb_model_v1.pkl model is trained with this version
 PyPDF2==3.0.1
-xgboost==2.0.3
+xgboost==1.4.0
+google-cloud-aiplatform>=0.8.0
+google-cloud-texttospeech
```
Comment (SHINE-six) [inline backend/requirements.txt:26]: which version

File: backend/requirements.txt [hunk 5]
```diff
@@ -21,7 +21,10 @@ gensim==4.3.2
...
+
```
Comment (SHINE-six) [inline backend/requirements.txt:28]: i believe this also need
vertexai==1.48.0

File: backend/server.py [hunk 6]
```diff
@@ -280,10 +282,9 @@ async def create_session(sessionJson: Request):
 
 # ----------------------- Audio thingy -----------------------
 def text_LLM_tts_wavToJson(userTranscript: str):
-    outputText = LLM.main(userTranscript)
-    print(mongoDB.appendDataToDocument("conversationLog", {"user": "Ai - EVA", "text": outputText['reply'], "flag": outputText['flag']}, uniqueSessionID))
-    print(outputText)  # to remove
-    tts.main(outputText['reply'])
+    outputText = LLM_copy.main(userTranscript)
+    print(mongoDB.appendDataToDocument("conversationLog", {"user": "Ai - EVA", "text": outputText['reply']}, uniqueSessionID))
+    googleTTS.main(outputText)
```
Comment (SHINE-six) [inline backend/server.py]: googleTTS.main(outputText['reply'])

**Review by SHINE-six — CHANGES_REQUESTED** [review]

File: backend/server.py (hunk 6 above)
Comment (chenming7777) [reply backend/server.py]: done

File: backend/requirements.txt (hunk 5 above)
Comment (chenming7777) [reply backend/requirements.txt:28]: done

File: backend/requirements.txt (hunk 4 above)
Comment (chenming7777) [reply backend/requirements.txt:26]: dunno why still here

File: backend/mbti_test.py (hunk 3 above)
Comment (chenming7777) [reply backend/mbti_test.py:89]: commented not an issue dulu

File: backend/mbti_test.py (hunk 2 above)
Comment (chenming7777) [reply backend/mbti_test.py:87]: commented not an issue dulu

File: backend/googleTTS.py (hunk 1 above)
Comment (chenming7777) [reply backend/googleTTS.py]: done

**Pull request closed without merging**